tools/ci/pr_preview.py

```diff
@@ -123,6 +123,26 @@ def get_pull_requests(self, updated_since):
 
         return data['items']
 
+    @guard('core')
+    def pull_request_is_from_fork(self, pr_number):
```

Comment:
This is a bit inconsistent with other APIs, which all take the `pull_request` dict instead of `pr_number`. I'd suggest using `pull_request` here, too.